src/modules/layout/containers/LocationLayout.js

```diff
@@ -64,20 +97,19 @@ export class LocationLayout extends React.Component<PropsType, StateType> {
                                            isExtrasEnabled={cityModel.extrasEnabled}
                                            onStickyTopChanged={this.onStickyTopChanged} />}
                    footer={<LocationFooter city={city} language={language} />}
-                   toolbar={showCategoriesToolbar && <CategoriesToolbar city={city}
-                                                                        language={language}
-                                                                        pathname={pathname}
-                                                                        categories={categories} />}>
-        {children}
+                   toolbar={showCategoriesToolbar && cities && <CategoriesToolbar categories={categories}
+                                                                                  location={location} />}>
+
+        <React.Fragment>
+          {children}
+          {this.renderFeedbackModal()}
+        </React.Fragment>
       </Layout>
```

Comment:
That way, the header is displayed above the feedbackmodal. To fix that do the following:
here: 
```jsx
<React.Fragment>
      <Layout asideStickyTop={this.state.asideStickyTop}
                   header={<LocationHeader isEventsEnabled={cityModel.eventsEnabled}
                                           isExtrasEnabled={cityModel.extrasEnabled}
                                           onStickyTopChanged={this.onStickyTopChanged} />}
                   footer={<LocationFooter city={city} language={language} />}
                   toolbar={showCategoriesToolbar && cities && <CategoriesToolbar categories={categories}
                                                                                  location={location} />}>

          {children}
      </Layout>
      {this.renderFeedbackModal()}
    </React.Fragment>
```

Plus some add. changes (see below)

Reply:
Attention: If you do this, you need to inject the font-families for the feedbackModal again...
Maybe it would be nice, to extend the Layout component to allow for Modals...

Reply:
Like with toolbar, header and stuff?

Reply:
Right, I thought I checked it. :/